### ISSUE TITLE
Excavation Sites wont unlock

### ISSUE THREAD

**Issue by Jynnn** [PrehistoricKingdom/feedback#506]
So the La Brea Tar Pit excavation site says it requires 2 stars and $250,000 to unlock it, for example.

I have both, but it is still red and locked.

This is the same for other 2 star required excavation sites

![20220427135756_1](https://user-images.githubusercontent.com/54244258/165629795-03bc30d4-8180-4734-80d3-915ec730a86b.jpg)
![20220427135804_1](https://user-images.githubusercontent.com/54244258/165629799-0229e726-167d-49a2-8d23-91831b76e089.jpg)

**Comment by WhyWatt** [comment]
Yeah I have the same issue here - went with the Spain map if that helps any

**Comment by Jynnn** [comment]
i should clarify that i too was on Spain map

**Comment by Delitryx** [comment]
I had this problem on Costa Rica, restarting the game fixed it.
